backend/db/seeds/20180415220000-seed-demo.js

```diff
@@ -8,7 +8,7 @@ import moment from 'moment'
 const imageUploadDir = 'uploads/images'
```

Comment:
This was me hot fixing the one thing. Might go away since I'll merge master into this branch before final review